apps/web/src/components/WalletCreateDrawer.tsx

```diff
@@ -49,73 +63,120 @@ const WalletCreateDrawer: React.FC<WalletCreateDrawerProps> = ({
     onClose();
   };
 
+  const requestClose = useCallback(() => {
```

Comment:
Your requestClose logic is solid, but renaming it to handleCloseRequest reads more in line with handler naming conventions.